packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -165,7 +154,7 @@ export function LinkUI( props ) {
 	const link = {
 		url: props.link.url,
 		opensInNewTab: props.link.opensInNewTab,
-		title: props.link.label && navStripHTML( props.link.label ),
+		title: props.link.label && stripHTML( props.link.label ),
```

Comment:
Noting for posterity that I believe `navStripHTML` was intended as a short term work around for the fact that `stripHTML` originally also collapsed leading/trailing whitespace in a given string (due to implementation details).

Whilst that was being resolved I implemented `navStripHTML` which has nearly identical implementation except it does _not_ run `safeHTML` over the string. Hopefully that is just an oversight and not a huge problem.

So if there are suddenly bugs with escaping in the Link UI then we know to take a good look at this change.